modules/cluster-logging-log-forwarding-configure.adoc

```diff
@@ -96,12 +96,72 @@ spec:
 * Enter a name for the pipeline
 * Specify the source type: `logs.app`, `logs.infra`, or `logs.audit`.
 * Specify the name of one or more outputs configured in the CR.
-
++
 [NOTE]
 ====
 If you set `disableDefaultForwarding: true` you must configure a pipeline and output for all three types of logs, application, infrastructure, and audit. If you do not specify a pipeline and output for a log type, those logs are not stored and will be lost.
 ==== 
 
+== Updating the Log Forwarding Custom Resource
+
+If you make changes to the log forwarding outputs or pipelines, you need to disable log forwarding, edit the CR, then enable log forwarding again.
```

Comment:
This really should be unnecessary.  If you have to do this then there is a bug in the code.

Reply:
@mburke5678  Yes, It is unnecessary.  The change can be apply without disbale log forwarding. if the forward didn't take effect,  we will filed a bug to trace it.

Reply:
@anpingli Thanks. I think I misunderstood a comment you made in Slack. I will follow up with https://github.com/openshift/openshift-docs/pull/20675 to remove this section,